iphone/Classes/TiUIView.h

```diff
@@ -279,6 +279,7 @@ void ModifyScrollViewForKeyboardHeightAndContentHeightWithResponderRect(UIScroll
 - (void)processTouchesMoved:(NSSet *)touches withEvent:(UIEvent *)event;
 - (void)processTouchesEnded:(NSSet *)touches withEvent:(UIEvent *)event;
 - (void)processTouchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event;
+-(void)processKeyPressed:(id)key;
```

Comment:
Missing space, also you can use an `NSString` type here, because it's not on proxy-level and only can be a String as far as I know